pyresample/geometry.py

```diff
@@ -1822,7 +1822,8 @@ def get_lonlats(self, nprocs=None, data_slice=None, cache=False, dtype=None, chu
             from dask.array import map_blocks
             res = map_blocks(invproj, target_x, target_y,
                              chunks=(target_x.chunks[0], target_x.chunks[1], 2),
-                             new_axis=[2], proj_dict=proj_def)
+                             new_axis=[2], proj_dict=proj_def,
+                             dtype=dtype)
```

Comment:
What version of pyproj did you test this with? I noticed this recently too, but dask should be checking the dtype for us automatically. Are you seeing something different?

Reply:
See #242 and see it failing in Travis CI with [this commit](https://travis-ci.org/pytroll/pyresample/builds/623098241?utm_source=github_status&utm_medium=notification), as I committed and pushed the test before I did so for the fix.